### DISCUSSION TITLE
Details for Network TIme Sync Implementation

### DISCUSSION THREAD

**Discussion by bertoldi-silabs** [openthread/openthread#8171] [Q&A]
Hello, I am evaluating the effort needed to implement the network time sync feature for ot-efr32.

Can somebody point me to any information regarding the functionality of this feature, or what is needed from a platform to be able to support a network in which OPENTHREAD_CONFIG_TIME_SYNC_ENABLE is configured? As of now, I am seeing issues with a child node trying to join a network and respond to a Child ID Request with this option enabled, but I am interested in supporting the time sync aspects as well.

Any pointers would be appreciated. Thanks you!

**Comment by suveshpratapa** [comment]
@jwhui @abtink Can we get some pointers on this question? We are trying to test this non-spec feature, so any documentation on how network time-sync is supposed to work (such as message handshakes etc) would be helpful.
